src/plugins/ovp/components/video-screen/VideoPreviewCard.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Card, CardHeader } from 'reactstrap';
+import useNode from '@triniti/cms/plugins/ncr/components/useNode.js';
+import damUrl from '@triniti/cms/plugins/dam/damUrl.js';
+import artifactUrl from '@triniti/cms/plugins/ovp/artifactUrl.js';
+
+export default function VideoPreviewCard(props) {
```

Comment:
@rsumilang we don't need a node here, no useNode.  just taking the nodeRef you get, get the id from it and pass it to dam/components/asset-preview/index.js